### DISCUSSION TITLE
PP365 Hjem side på Portefølje - Forbedre opplevelsen for Gjester

### DISCUSSION THREAD

**Discussion by pzljanb** [Puzzlepart/prosjektportalen365#1027] [General]
### Beskriv feilen

Når man får tilgang til PP365 som gjest og går til fremsiden og ser på prosjektutlistningswebparten oppleves denne som at den feiler.
Årsaken til dette er at vertikalen "Mine prosjekter" ikke inneholder noen prosjekter selv om gjesten er medlem i prosjekter.

### Hvordan reprodusere feilen

Trinn for å gjenskape:

 1. Gå til PP365 som Gjest
( Påse at du har tilgang i minium ett prosjekt)
 3. Klikk på vertikalen "Mine prosjekter" og observer at listen er tom
 4. Gå til vertikalen "Tilgang til" og observer at prosjektet finnes der.

### Forventet oppførsel

Gjesten må enten oppleve at vertikalen "Mine prosjekter" inneholder da faktiske "mine prosjektene" eller så må vertikalen skjules for brukere av typen Gjest.

### Skjermbilder

![image](https://user-images.githubusercontent.com/58074707/220861240-29bfca2d-f2a6-4db8-aa0e-602e61392c82.png)
![image](https://user-images.githubusercontent.com/58074707/220861301-864f29f4-a3c8-4212-9d15-1b6709d73881.png)
![image](https://user-images.githubusercontent.com/58074707/220861363-f0af2259-4164-455e-82f0-4d36324a36fb.png)

### Ytterligere informasjon

Problemet kan skyldes at en Gjest ikke har tilgang til Graph API. Det å gi en Gjest tilgang kan i så tilfelle være løsningen. Men i så tilfelle må vi opprette en type forutsetning og/ eller forklaring som følger med til kunde i et slikt tilfelle.

### Hvilke nettleser(e) oppleves feilen på?

Microsoft Edge

### Versjon

1.8.0

**Comment by okms** [comment]
Vi mangler nok tilgang til å berike prosjektene skikkelig for gjestebrukere. Jeg vil foreslå at vi skjuler "Mine prosjekter" for gjestebrukere. Det å basere seg på at man må heve tilgangsnivået for gjestebrukere i hele tenanten er for stor inngripen til at jeg synes det er forsvarlig å anbefale det, men det er en mulig workaround som vi kan teste

**Comment by okms** [comment]
For test-tenanten var gjeste-rettighetene satt på det mest restriktive nivået, vi tester med standardinstillingene og ser om det har noen effekt. Det kan sikkert ta noen timer før dette tar effekt 
![CleanShot 2023-02-23 at 11 10 04](https://user-images.githubusercontent.com/170147/220878046-92e92348-66ca-4133-97da-f8362a44dd0e.png)

**Comment by pzljanb** [comment]
Dette fungerer nå @olemp & @okms 
![image](https://user-images.githubusercontent.com/58074707/220891972-719eb840-2f68-4c22-a446-f2f8d85c9b44.png)



Jeg foreslår at vi for dette Issuet kun lager en beskrivelse som vi legger ved her på GitHub.
Og forklarer dette til de som er brukere av PP365 :-)

**Comment by olemp** [comment]
> Dette fungerer nå @olemp & @okms ![image](https://user-images.githubusercontent.com/58074707/220891972-719eb840-2f68-4c22-a446-f2f8d85c9b44.png)
> 
> Jeg foreslår at vi for dette Issuet kun lager en beskrivelse som vi legger ved her på GitHub. Og forklarer dette til de som er brukere av PP365 :-)

Oppdaterer du issue tittel og beskrivelse til å korrespondere med dette? (Så slipper man lese gjennom alle kommentarene for å forstå hva som skal gjøres.)
